pktfwd/manager.go

```diff
@@ -233,30 +233,53 @@ func (m *Manager) statusRoutine(bgCtx context.Context, errC chan error) {
 	}
 }
 
+func (m *Manager) networkRoutine(bgCtx context.Context, errC chan error) {
+	err := m.netClient.RefreshRoutine(bgCtx)
+	if err != nil {
+		errC <- errors.Wrap(err, "Couldn't refresh account server token")
+	}
```

Comment:
I meant make an error channel here, and return `chan error`. This function is owner of that channel and controls open, close, sending and capacity